### PR TITLE
Add support for the PiVoyager UPS pHAT

### PR DESCRIPTION
**Pull request by omzlo**

### CODE REVIEW

File: draft/overlay/pivoyager.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    device: STM32F030C6T6
+-->
+#PiVoyager UPS
+
+**The PiVoyager is a UPS with a programmable watchdog, wake-up and a real-time calendar for the Raspberry-Pi.**
+
+The **PiVoyager** is uninterruptible power supply (UPS) for the Raspberry Pi designed to work with standard Li-Ion or LiPo batteries, featuring a programmable watchdog, automatic restart, and a real-time calendar. 
+The **PiVoyager** is designed as a Pi Zero pHAT, but works on any Raspberry-Pi with a 40 pin header, including the Raspberry Pi 3B+ and 4. 
+
+When the **PiVoyager** is plugged to a USB power source (USB micro-B) and a Li-Ion/LiPo battery, it will both power your Raspberry Pi and charge the battery if needed, with a selectable charge current of 1000mA (default) or 500mA. If USB power is removed, the **PiMaster** automatically switches to the battery and continues to power the Raspberry Pi at 5V (2.1A max), thanks to a boost converter.
```
Comment (lurch) [inline draft/overlay/pivoyager.md]: PiMaster ?

Comment (omzlo) [reply draft/overlay/pivoyager.md]: Oups. Sorry this is a typo. I fixed it in a new commit.

**Comment by omzlo** [comment]
Let me know if you need anything else to add this board to pinout.xyz. This website is a great initiative!

**Comment by Gadgetoid** [comment]
Will check over this, thank you! Turns out the thing I need most is... as always... time!

**Pull request merged**